packages/block-editor/src/hooks/block-bindings.js

```diff
@@ -159,7 +98,94 @@ function EditableBlockBindingsPanelItems( {
 	fieldsList,
 } ) {
 	const { updateBlockBindings } = useBlockBindingsUtils();
-	const isMobile = useViewportMatch( 'medium', '<' );
+	const [ isOuterModalOpen, setOuterModalOpen ] = useState( false );
+	const dataSet = Object.entries( fieldsList[ 'core/post-meta' ] ).map(
+		( [ key, field ] ) => {
+			const value =
+				field.value === undefined
+					? `${ field.label } value`
+					: field.value;
+
+			return {
+				id: key,
+				label: field.label || key,
+				value: value !== '' ? value : `Add a new ${ field.label } `,
+			};
+		}
+	);
+
+	const fields = [
+		{
+			id: 'label',
+			label: 'Label',
```

Comment:
I'd like to make the `type` required soon. There're some fallbacks for things such as sorting to work in most common cases (text, number, date), but it'd be great to be more explicit here by providing `type: text`.